include/ceed/ceed.h

```diff
@@ -119,15 +119,20 @@
 #  endif
 #endif
 
-#include <stdint.h>
 #include <stdarg.h>
-#include <stdio.h>
 #include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
 
 /// Integer type, used for indexing
 /// @ingroup Ceed
 typedef int32_t CeedInt;
 
+/// Integer type, used array sizes
+/// @ingroup Ceed
+typedef ptrdiff_t CeedSize;
```

Comment:
Should it be named like this because `size_t` is unsigned and this is signed? On the other hand, this is like Rust's `isize`. No strong preference, but want the decision to be deliberate.
```suggestion
typedef ptrdiff_t CeedCount;
```

Reply:
I was thinking size like isize but I'm fine with either.

Reply:
That'll smooth things out when we change the library core to Rust. :smile: